app/models/github_organization.rb

```diff
@@ -107,6 +107,10 @@ def create_organization_webhook(config: {}, options: {})
     end
   end
 
+  def ping_organization_webhook(webhook_id)
+    @client.ping_org_hook(@id, webhook_id)
+  end
+
   def remove_organization_webhook(webhook_id)
     return unless webhook_id.present?
     @client.remove_org_hook(@id, webhook_id)
```

Comment:
Just realized we aren't doing it here and I missed it, mind wrapping this up as well?